facebook/sapp
# Adds jest tests for Traces.js

Adds jest tests for Traces.js file in Traces.test.js with jest
snapshots. Modifies Source and Traces file to make the components
importable from the test file.

Test Plan
- checkout the PR
- `cd sapp/ui/frontend`
- Run `npm run-script ui-tests`
- See the jest tests

Screenshots:
<img width="1440" alt="Screenshot 2021-10-14 at 3 00 07 PM" src="https://user-images.githubusercontent.com/8947010/137291149-6b89991c-34ad-4221-b9fa-3190d79e923e.png">

Signed-off-by: Abishek V Ashok <abishekvashok@gmail.com>
